### ISSUE TITLE
Error: Redis WATCH detected a conflicting update

### ISSUE THREAD

**Issue by clouserw** [mozilla/fxa#2999]
https://sentry.prod.mozaws.net/operations/auth-prod/issues/6489019/

```
Error: Redis WATCH detected a conflicting update
  File "/app/lib/error.js", line 992, in Function.AppError.redisConflict
    return new AppError({
  File "/app/lib/redis.js", line 25, in Object.object.(anonymous function) [as update]
    throw error.redisConflict();
```

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-460)
┆Issue Number: FXA-460

**Comment by clouserw** [comment]
@dannycoates we're seeing this occur in low levels since the push.  Any thoughts?

**Comment by rfk** [comment]
>   File "/app/lib/redis.js", line 25, in Object.object.(anonymous function) [as update]

AFAICT this only caller of `redis.update()` is in `./lib/db.js`, where we manage some session-token data in redis.  In theory we should only see this conflict if a user tries to perform two concurrent actions that modify sessionToken metadata, e.g. by syncing two different devices at the same instant.

I don't have a theory about what could have changed in the latest release to cause this to spike, and I'm lightly suspicious that it may indicate some concurrency bug in the server code, where the server itself is trying to do concurrent modifications as part of handling a single request.

**Comment by clouserw** [comment]
@vbudhram is adjusting the logging to log a better stack

**Comment by dannycoates** [comment]
I've reproduced this locally. The browser is making two calls to `/certificate/sign` in quick succession. Now I'm trying to figure out what changed to trigger the conflict.

**Comment by dannycoates** [comment]
I also reproduced with v1.147.5

**Comment by dannycoates** [comment]
ima chalk this up to updating our sentry dependency so now fewer errors are getting dropped

**Comment by rfk** [comment]
Could be fallout from recent refactoring in the browser code, I'll cross-link this in the sync decouple channel.

**Comment by rfk** [comment]
> I've reproduced this locally. The browser is making two calls to /certificate/sign in quick succession. 

I haven't been able to reproduce this in latest nightly, and I wonder if we actually perturbed it *away* with the latest refactor to how FxA works in the browser.  @dannycoates are you still able to reproduce?

**Comment by dannycoates** [comment]
fixed by #3278
